index.js

```diff
@@ -128,21 +146,18 @@
    */
   EventTracker.prototype.send = function send(done) {
     if (this.buffer.length) {
-      var data = JSON.stringify(this.buffer);
-
-      var hash = this.calculateHash(this.clientSecret, data);
+      if (this.debug) {
+        return logEvent(this.buffer);
+      }
 
-      var headers = {
-        'Content-Type': 'text/plain',
-      };
+      var url = this.endpoint + '?key=' + this.key + '&mac=' + this.calculateHash(this.secret, data);
```

Comment:
good call on encoded, and yes it's kind of a pain to have to handle the query with `fetch` when we could do it in the lib for everyone.